docs/rules/no-misspelled-operator.md

```diff
@@ -0,0 +1,24 @@
+# no-misspelled-operator
+
```

Comment:
@inverno we need to update RSPEC accordingly (probably we can do it as JS subtask)